src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -50,24 +49,9 @@ public partial class NodeViewModel : ViewModelBase
 
         public NodeModel NodeModel { get { return nodeLogic; } private set { nodeLogic = value; } }
 
-        public bool IsFullyConnected
-        {
-            get { return isFullyConnected; }
-            set
-            {
-                isFullyConnected = value;
-                RaisePropertyChanged("IsFullyConnected");
-            }
-        }
-
         public LacingStrategy ArgumentLacing
         {
             get { return nodeLogic.ArgumentLacing; }
-            set
-            {
-                nodeLogic.ArgumentLacing = value;
-                RaisePropertyChanged("ArgumentLacing");
-            }
```

Comment:
`NodeViewModel` should not allow property modifications such as these, they now become read-only properties.